docs/developer_handbook.md

```diff
@@ -82,3 +82,35 @@ You'll always need to add support for new records in the ES exporter. Even if yo
 5. Document this new filter option in the elasticsearch exporter's [README](../exporters/elasticsearch-exporter/README.md).
 6. Add a mapping for the ValueType to the [TestSupport](../exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/TestSupport.java).
 
+## How to do inter-partition communication?
+
+Generally, each [partition](https://docs.camunda.io/docs/next/components/zeebe/technical-concepts/partitions/) is isolated from the others. The gateway chooses the partition for each user command and sends the command to that partition's leader. This distributes the load over the partitions. When the engine processes commands, the follow-up records are written on the same partition.
+
+In some cases, a partition needs to communicate with other partitions:
+- Deployment distribution: Each partition must know the related process and decision models. The deployment distribution ensures each partition knows the process and decision models.
+- Message subscription and correlation: messages are published on a specific partition and correlated to process instances on other partitions to keep messaging scalable. This requires the engine to open subscriptions on the relevant partitions.
+
+### How to send a command to another partition?
+
+The engine can send a command to another partition using `InterPartitionCommandSender`. Note that this may fail because network communication is not reliable. So for all inter-partition communication, you must use some retry mechanism.
+
+For example, the `DeploymentRedistributor` resends deployment distribution commands to other partitions.
+
+### How to process a command received from another partition?
+
+As we've seen, the sending partition may send a command many times to another partition. So, the engine must be able to deal with these redundant commands (i.e. commands that are sent many times). To be specific, the engine must process commands received from another partition idempotently.
+
+:::info
```

Comment:
:x: this is not rendered correctly (see https://github.com/camunda/zeebe/blob/korthout-doc-interpartition-communication/docs/developer_handbook.md)

Reply:
Interesting. I've found that [GH supports info blocks in the form of block quotes](https://github.com/community/community/discussions/16925) (beta feature that may be subject to change). 

That looks like this:
> **Note**
> So information, much wow!

There is [a discussion ongoing](https://github.com/community/community/discussions/16925#discussioncomment-2791869) about supporting the `:::info\n...\n:::`syntax.